test/other/metadce/test_metadce_files_wasmfs.size

```diff
@@ -1 +1 @@
-51960
+52403
```

Comment:
This is the cost of supporting nanoseconds, I suppose?

I am not sure it's worth it, personally. 500 bytes isn't a lot, but the benefit is unclear to me... but if you feel strongly @tlively I don't object.

Regardless, if we decide we want nanoseconds let's do it in its own PR?

Reply:
Yes, I think having better than second precision is worth 500 bytes. I'm surprised it costs that much, though.